app/config/settings_prod.py

```diff
@@ -45,7 +45,9 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#secure-proxy-ssl-header
 SECURE_PROXY_SSL_HEADER = ('HTTP_CLOUDFRONT_FORWARDED_PROTO', 'https')
 
-ALLOWED_HOSTS = []
+# We need to have the IP of the Pod/localhost in ALLOWED_HOSTS
+# as well to be able to scrape prometheus /metrics
+ALLOWED_HOSTS = [os.getenv('THIS_POD_IP', '127.0.0.1')]
```

Comment:
Why not adding this ip directly in the ALLOWED_HOSTS environment variable ?

Reply:
added it to the readme